### ISSUE TITLE
How can I load an array of images from local disk using SDWebImage?

### ISSUE THREAD

**Issue by HaileApp** [rs/SDWebImage#2212]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 11.0 / 10.13.0 / 11.0 / 4.0
 SDWebImage Version      | e.g. 4.2.0 / 4.1.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 9 / Xcode 8
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by HaileApp** [comment]
I want to use SDWebImage Library in my iOS app.I have more than 160 images found in the Assets folder.So how can I use it to load the images?

**Comment by dreampiggy** [comment]
Use [readOnlyCachePath](https://github.com/rs/SDWebImage/blob/master/SDWebImage/SDImageCache.h#L98-L104). Attention you should put the Assets folder not into .xcassets format(Xcode will compress it into a `assets.car` and could not load by disk cache) 

Our documentation in [Wiki Page](https://github.com/rs/SDWebImage/wiki) does not contains this feature, because this maybe advanced usage.

**Comment by HaileApp** [comment]
I can't get the code with Swift format..can you please give me the code snapshot? Here I did the images in a new Group and use this code and it works but i have no idea how it impacts the memory.
let bundlePath = Bundle.main.path(forResource: imageList[imageIndex], ofType: "jpg") imageView.sd_setImage(with: nil, placeholderImage: UIImage(contentsOfFile: bundlePath!))

**Comment by dreampiggy** [comment]
@HaileApp You can always get the Swift interface through Xcode's `Generated Interface`. The Swift interface is `open func addReadOnlyCachePath(_ path: String)`
